### PR TITLE
Allow update to action without requiring a code artifact.

### LINKED ISSUE #93: Allow updating action parameters without including action body
`wsk` cli allows setting new default parameters without updating the action body. This library treats the `action` parameter as mandatory for update operations. 

Need to update the `actions.update` method to support this.

### PR DESCRIPTION
**Pull request by rabbah**
Closes #93.

### CODE REVIEW

**Comment by rabbah** [comment]
@csantanapr can you review this?

**Comment by rabbah** [comment]
Test added. Pre path test results:

```
  1 test failed

  actions › update a action with no paramters

  /Users/rabbah/projects/openwhisk-client-js/lib/actions.js:94

   93:     if (!isCodeAction && !isSequenceAction) {                   
   94:       throw new Error(messages.MISSING_ACTION_OR_SEQ_BODY_ERROR)
   95:     }                                                           

  Error thrown in test:

  Error {
    message: 'Missing mandatory action or sequence parameter from options.',
  }

  Actions.actionBody (lib/actions.js:94:13)
  Actions.create (lib/actions.js:56:25)
  create (test/unit/actions.test.js:416:18)
```

File: test/unit/actions.test.js [hunk 1]
```diff
@@ -400,6 +400,22 @@ test('create a new action with annotations', t => {
   return actions.create({ name: '12345', action, annotations })
 })
 
+test('update a action with no paramters', t => {
```
Comment (shazron) [inline test/unit/actions.test.js]: nit: `paramters` spelling

**Comment by rabbah** [comment]
Thanks @shazron @purplecabbage @dgrove-oss.

**Pull request merged**